src/codecs/format/json.rs

```diff
@@ -80,13 +84,60 @@ impl From<&JsonDeserializerConfig> for JsonDeserializer {
     }
 }
 
+/// Config used to build a `JsonSerializer`.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+pub struct JsonSerializerConfig;
+
+impl JsonSerializerConfig {
+    /// Creates a new `JsonSerializerConfig`.
+    pub const fn new() -> Self {
+        Self
+    }
+}
+
+#[typetag::serde(name = "json")]
+impl SerializerConfig for JsonSerializerConfig {
+    fn build(&self) -> crate::Result<BoxedSerializer> {
+        Ok(Box::new(JsonSerializer))
+    }
+}
+
+/// Serializer that converts an `Event` to bytes by extracting the message key.
```

Comment:
This comment seems incorrect? Maybe copied from `text`?

Reply:
Oopa, good catch 👀